CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Add `BakeScreenreaderSpans` direction (minor)
 * Fix `BakeIndex` to group terms by character in polish books and transliterate it for others (minor)
-
+* Add `figure_reference` option for `BakeNoteInjectedQuestion` to bake exercise-context figure reference (minor)
```

Comment:
Don't forget to update the changelog to reflect the new change! I believe this change is `major` as single-part injected exercises should always be changed.